### PR TITLE
Migrate operator sdk 0.12

### PR DESCRIPTION
**Pull request by sabre1041**
Operator SDK 0.12 migration

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ sudo: required
 language: go
 go_import_path: github.com/redhat-cop/quay-operator
 go:
-- "1.12"
+  - "1.13"
```
Comment (kurtismullins) [inline .travis.yml:5]: Is golang 1.13 required to build? The main reason I ask is because Redhat's OSBS/build pipeline is still using 1.12.

Comment (sabre1041) [reply .travis.yml:5]: Yes. it is now required by operator-sdk 0.12

File: docs/storage.md [hunk 2]
```diff
@@ -218,23 +218,23 @@ spec:
       - name: rhocs
         rhocs:
           hostname: <hostname>
-          is_sucure: <is_secure>
-          access_key: <access_key>
-          secret_key: <secret_key>
-          bucket_name: <bucket_name>
+          sucure: <sucure>
```
[the remark this answers is not in the document]
Comment (sabre1041) [reply docs/storage.md]: Updated PR

**Review by kurtismullins — APPROVED** [review]
LGTM

**Review by kurtismullins — APPROVED** [review]

**Pull request merged**